src/carla_ros_bridge/ego_vehicle.py

```diff
@@ -161,8 +161,15 @@ def apply_control(self):
         vehicle_control.steer = self.info.output.steer
         vehicle_control.throttle = self.info.output.throttle
         vehicle_control.reverse = self.info.output.reverse
-        # send control command out
-        self.carla_actor.apply_control(vehicle_control)
+
+        # send control command out, if there is a ROS control publisher
+        ros_control_topic = rospy.get_published_topics(namespace=self.topic_name() + "/ackermann_cmd")
+        print(ros_control_topic)
+        ros_control_topic.extend(rospy.get_published_topics(namespace=self.topic_name() + "/vehicle_control_cmd"))
+        print(ros_control_topic)
```

Comment:
```suggestion
        # print(ros_control_topic)
```
same continiously printing empty list